src/app/resources/resources-add.component.ts

```diff
@@ -257,6 +262,22 @@ export class ResourcesAddComponent implements OnInit {
     this.disableDownload = false;
     this.disableDelete = false;
     this.resourceForm.updateValueAndValidity();
+
+    if (this.file.type === 'application/zip') {
```

Comment:
Please use the `simpleMediaType` method.  Some zip files have different MIME types than this.